plugins/CoreHome/Columns/VisitLastActionYear.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\CoreHome\Columns;
+
+use Piwik\Columns\DimensionMetricFactory;
+use Piwik\Columns\MetricsList;
+use Piwik\Plugin\Dimension\VisitDimension;
+use Piwik\Metrics\Formatter;
+
+class VisitLastActionYear extends VisitDimension
+{
+    protected $columnName = 'visit_last_action_time';
+    protected $type = self::TYPE_DATETIME;
+    protected $segmentName = 'visitEndServerYear';
+    protected $nameSingular = 'VisitTime_ColumnVisitEndServerYear';
+    protected $sqlSegment = 'YEAR(log_visit.visit_last_action_time)';
+    protected $acceptValues = '2016, 2017, 2018, ..., 9998, 9999';
+
+    public function __construct()
+    {
+        $this->suggestedValuesCallback = function ($idSite, $maxValuesToReturn) {
+            return range(2016, min(2036, $maxValuesToReturn));
```

Comment:
I'm not sure if that's what it's meant to be.  If `$maxValuesToReturn` is `30` like in the tests, it returns a range from 2016 down to 30. I guess it would be better to use something like `return range(2016, 2016 + $maxValuesToReturn);` ?

Reply:
Absolutely right, not sure what I was thinking... pushing a fix now.